### PR TITLE
ctpdev: removing dependencies on RunManager

### PR DESCRIPTION
**Pull request by lietava**
Removing dependency on RunManager as discussed in
https://github.com/AliceO2Group/AliceO2/pull/13224
I left old lines commented as it may be moved back after O2 update.

### CODE REVIEW

**Comment by lietava** [comment]
Hi @knopers8 @Barthelemy , please merge or comment.

**Comment by Barthelemy** [comment]
@lietava We might take a couple of days to review, but we will do it, no worries.

File: Modules/CTP/src/RawDataQcTask.cxx [hunk 1]
```diff
@@ -80,13 +80,27 @@ void CTPRawDataReaderTask::startOfActivity(const Activity& activity)
...
+  // o2::ctp::CTPRunManager::setCCDBHost(ccdbName);
   bool ok;
-  o2::ctp::CTPConfiguration CTPconfig = o2::ctp::CTPRunManager::getConfigFromCCDB(mTimestamp, run, ok);
+  // o2::ctp::CTPConfiguration CTPconfig = o2::ctp::CTPRunManager::getConfigFromCCDB(mTimestamp, run, ok);
+  auto& mgr = o2::ccdb::BasicCCDBManager::instance();
+  mgr.setURL(ccdbName);
+  map<string, string> metadata; // can be empty
+  metadata["runNumber"] = run;
+  auto ctpconfigdb = mgr.getSpecific<o2::ctp::CTPConfiguration>(o2::ctp::CCDBPathCTPConfig, mTimestamp, metadata);
+  if (ctpconfigdb == nullptr) {
```
Comment (Barthelemy) [inline Modules/CTP/src/RawDataQcTask.cxx:91]: the logic with the `ok` seems circumvoluted...

**Review by Barthelemy — APPROVED** [review]
Please consider my comments. i understand that you need to merge it but perhaps you can make a second PR to fix that.

**Comment by lietava** [comment]
@Barthelemy thanks for your comments and merge. I will try to implement comments on the next occasion.

**Pull request merged**